src/genie/libs/parser/iosxe/show_device_tracking.py

```diff
@@ -163,4 +163,122 @@ def cli(self, output=None):
         return device_tracking_database_dict
 
 
+# ======================================
+# Schema for:
+#  * 'show device-tracking database int'
+# ======================================
+class ShowDeviceTrackingDatabaseIntSchema(MetaParser):
+    """Schema for show device-tracking database int."""
```

Comment:
Can you please add  {interface} in command?

Reply:
Done!